tests/L0/run_amp/test_update_scale_hysteresis.py

```diff
@@ -0,0 +1,105 @@
+import unittest
+import random
+import math
+
+from apex import amp
+import torch
+
+from utils import common_init
+
+try:
+  import amp_C
+  from amp_C import update_scale_hysteresis
+  disabled = False
+except ImportError as err:
+  print("amp_C fused kernels unavailable, disabling TestUpdateScaleHysteresis.  ImportError was ", err)
+  disabled = True
+
+def isfinite(val):
+    return ((val >= torch.finfo(torch.float32).smallest_normal) and (val <= torch.finfo(torch.float32).max))
+
+class TestUpdateScaleHysteresis(unittest.TestCase):
+
+    def setUp(self):
+        common_init(self)
```

Comment:
this doesn't sound necessary to me as per https://github.com/NVIDIA/apex/blob/2386a912164b0c5cfcd8be7a2b890fbac5607c82/tests/L0/run_amp/utils.py#L15